src/components/DownloadComponents/UpcomingReleases/UpcomingReleasesItem/index.tsx

```diff
@@ -22,12 +28,28 @@ const UpcomingReleasesItem = ({
     [styles.itemToBeReleased]: !alreadyReleased,
   });
 
+  const releaseTitle = () => {
```

Comment:
This `if/if/if` sounds a little bit exhaustive, and it feels like there's some code duplication here.

For example, there's no need to call the `<FormattedMessage>` for every case. You can dynamically pass the translation IDs on the `id` property.

Reply:
Yeah I will test with this proposal

Reply:
You can create an enum to get the `id` based on the `releaseType`, by that way we can eliminate this function and use `<FormattedMessage />` directly inside the render